### DISCUSSION TITLE
Cannot run the project

### DISCUSSION THREAD

**Discussion by WasifRaza123** [mikaelacaron/Basic-Car-Maintenance#113] [General & Questions] [answered]
Hey @mikaelacaron, I have followed the instructions in the getting started tab. But I cannot run the project in simulator as I don't have a paid apple developer account. 
Error Screenshot: 

<img width="641" alt="Screenshot 2023-10-09 at 12 36 42 PM" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/67188463/179fda62-b101-41bd-a2d8-4593deca55c5">

**Comment by mikaelacaron** [comment]
@WasifRaza123 Can you change the bundle ID and just write your name rather than the `mycompany` part
Also are you trying to run on iOS?

**Reply by WasifRaza123** [reply]
I have changed the bundle id and tried but the same issue occurs. I am only trying to run on Xcode simulator.
<img width="1512" alt="Screenshot 2023-10-09 at 10 36 53 PM" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/67188463/c76b9576-a5c8-4e67-9038-823281402a3e">

**Comment by mikaelacaron** [comment]
You need to change the bundle ID using the `xcconfig` file
I would try to delete your local copy, and clone it again, following the steps

Also you can take a screen recording with Quicktime, and upload it to Dropbox or Google Drive, with a public link, and drop that here, and I can possibly help show you what's wrong, if there's still an error

**Reply by WasifRaza123** [reply]
Video Link:
https://drive.google.com/file/d/1ixJaSluY5TQlM4nTfyZ--5XrxX1Kx7qc/view?usp=share_link

**Reply by mikaelacaron** [reply]
Thank you for doing that! That really helped!

You are building for "My Mac" not for an iOS simulator

Also don't use an email for the dev team, if you don't have an ID just leave it from the template and let me know what happens

**Reply by WasifRaza123** [reply]
Hi @mikaelacaron, When building first time, getting the following errors:
<img width="1563" alt="Screenshot 2023-10-10 at 9 55 40 AM" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/67188463/fd3017ec-1d05-4769-b797-6ef1871bf4e6">

**Reply by mikaelacaron** [reply]
Can you choose an actual simulator? Rather than "any iOS device"

And that's all you've changed and this still happens?

**Reply by WasifRaza123** [reply]
Same issue after changing the simulator.

<img width="1563" alt="Screenshot 2023-10-10 at 8 49 40 PM" src="https://github.com/mikaelacaron/Basic-Car-Maintenance/assets/67188463/ae9f82ac-0a35-4cc3-b0ca-002cf509e726">

**Reply by mikaelacaron** [reply]
and this was after properly copying the steps from the README, you only changed the target to iOS?
Have you cleaned the project and deleted derived data, and tried to build again?

**Reply by WasifRaza123** [reply, marked as answer]
Sorry for the time wasted.
The problem got solved, I was using Mac OS 14.

**Reply by mikaelacaron** [reply]
That's okay! I'm glad you got it resolved!
